logigator-editor/src/app/components/work-area/work-area.component.html

```diff
@@ -4,9 +4,10 @@
 		<ng-container *ngFor="let project of allProjects">
 			<div class="tab" #tab
 			     *ngIf="!isSimulationMode || project.type === 'project'"
-			     [class.tab-active]="project.id === activeView?.project.id">
+			     [class.tab-active]="project.id === activeView?.project.id"
+				 (auxclick)="tabCloseClicked(project, $event)" >
```

Comment:
This fires on every mouse button except left click. You have to explicitly check for middle mouse button.